apps/price_pusher/src/sui/sui.ts

```diff
@@ -488,18 +488,24 @@ export class SuiPricePusher implements IPricePusher {
           options: { showEffects: true },
         });
       } catch (e) {
+        console.log("Merge transaction failed with error:");
+        console.log(e);
+        console.log((e as any).data);
+        console.log(JSON.stringify(e));
         if (
           String(e).includes(
             "quorum of validators because of locked objects. Retried a conflicting transaction"
           )
         ) {
+          /*
           Object.values((e as any).data).forEach((lockedObjects: any) => {
             lockedObjects.forEach((lockedObject: [string, number, string]) => {
               lockedAddresses.add(lockedObject[0]);
             });
           });
           // retry merging without the locked coins
           i--;
+           */
```

Comment:
something in this block above is what's throwing right now. I think it's ok to actually skip this entire batch of coins in the merging process though, so let's comment it out and see if that fixes the issue.